assemblies/core/lib/pom.xml

```diff
@@ -465,6 +466,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
```

Comment:
We usually don't include jdbc driver implementations here-- there tend to be license issues with distribution.  If needed for a test you can just scope it.